src/app/receptionist/shift.cpp

```diff
@@ -0,0 +1,12 @@
+#include "shift.hpp"
+
+namespace
+{
+static std::map<Shift, std::string> shift_text{{Shift::Morning, "morning shift"},
```

Comment:
might be an overkill in this particular example

Reply:
Changed